lms/models/lti_user.py

```diff
@@ -27,6 +28,9 @@ class LTIUser:
     application_instance_id: int
```

Comment:
We add the instance on top of the ID

The ID gets serialized over the network. The object only it's used on the app.